### PR TITLE
Export default template

### LINKED ISSUE #951: export DefaultTemplate or add to the registry object
### Prerequisites

- [x] I have read the [documentation](https://github.com/mozilla-services/react-jsonschema-form/blob/master/README.md#readme);
- [ ] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.

### Description
It would be really helpful if `DefaultTemplate` was exported or added to the [registry object](https://github.com/mozilla-services/react-jsonschema-form#the-registry-object). As far as I can tell, it's impossible to use the `DefaultTemplate` in a custom field.

My preference would be to ensure that `registry.FieldTemplate` has a default value (which would be `DefaultTemplate`.

### Steps to Reproduce
1. Create a custom field.
2. Try to wrap it in the `DefaultTemplate`.

#### Expected behavior
Be able to wrap custom fields in the `DefaultTemplate` or the defined custom field template.

#### Actual behavior
Cannot access the `DefaultTemplate`.

### Version
`1.0.2`

### PR DESCRIPTION
**Pull request by davidbarratt**
### Reasons for making this change
Fixes #951

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've checked the rendering of the Markdown text I've added
  - [ ] If I'm adding a new section, I've updated the Table of Content
* [x] **I'm adding or updating code**
  - [ ] I've added and/or updated tests
  - [ ] I've updated docs if needed
  - [x] I've run `npm run cs-format` on my branch to conform my code to [prettier](https://github.com/prettier/prettier) coding style
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ export function getDefaultRegistry() {
   return {
     fields: require("./components/fields").default,
     widgets: require("./components/widgets").default,
+    FieldTemplate: require("./components/templates/DefaultTemplate").default,
```
Comment (knilink) [inline src/utils.js]: https://github.com/mozilla-services/react-jsonschema-form/blob/e0192b75d24024370d52a43293931c8ddd769752/src/components/Form.js#L162

I think the form doesn't actually receive the template.

File: src/components/fields/SchemaField.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ import {
   deepEquals,
   getSchemaType,
 } from "../../utils";
+import DefaultTemplate from "../templates/DefaultTemplate";
```
Comment (knilink) [inline src/components/fields/SchemaField.js]: I think it's not necessary to import it here if we decided to pass it from the registry

File: src/utils.js (hunk 1 above)
Comment (epicfaace) [reply src/utils.js]: You're right -- @davidbarratt can you fix this?

File: src/components/fields/SchemaField.js (hunk 2 above)
Comment (epicfaace) [reply src/components/fields/SchemaField.js]: I agree

**Comment by epicfaace** [comment]
@davidbarratt just bumping this, can you please make the changes requested so I can merge this?

**Comment by epicfaace** [comment]
@davidbarratt bump?

**Comment by davidbarratt** [comment]
I don't use this library anymore, so feel free to take over the PR or close it. :)

**Comment by epicfaace** [comment]
Okay, I'll just close it, then.

**Pull request closed without merging**